src/sentry/monitors/tasks.py

```diff
@@ -205,39 +205,47 @@ def check_missing(current_datetime: datetime):
     )
     metrics.gauge("sentry.monitors.tasks.check_missing.count", qs.count(), sample_rate=1.0)
     for monitor_environment in qs:
-        try:
-            logger.info(
-                "monitor.missed-checkin", extra={"monitor_environment_id": monitor_environment.id}
-            )
+        check_missing_environment.delay(monitor_environment.id)
```

Comment:
not a celery expert but what's the overhead of a task here? would it make sense at all to batch them? 

is the overall goal of this PR simply to increase parallelism?

Reply:
Right now we're only at a few hundred but could always switch to chunks later
https://docs.celeryq.dev/en/latest/userguide/canvas.html#chunks